### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by ai1361720220000** [thu-ml/GFT#4]
In this project, i see the F.cross_entropy() is used for caclulating the loss, however mse loss is introduced in the paper.
i'm confused why it is different and does cross_entropy is more effective?

**Comment by ChenDRAG** [comment]
@ai1361720220000   Sorry for the delayed reply. Please see Sec 3.3 in the paper.  cross_entropy() loss is used for AR models,   MSE loss is used for diffusion models.     We will release our diffusion codebase soon. @ceku7
